libcontainer/process_linux.go

```diff
@@ -203,6 +204,28 @@ func (p *setnsProcess) start() (retErr error) {
 
 	// Get the "before" value of oom kill count.
 	oom, _ := p.manager.OOMKillCount()
+	useClone3 := false
+	if cgroups.IsCgroup2UnifiedMode() && p.initProcessPid != 0 {
```

Comment:
start() is already complex, let's move this to a function with a clear name, so it's more readable.